distribution/springboot_setup/receiver/application.properties

```diff
@@ -1,3 +1,11 @@
+# Base Url of a running openmrs reciever instance whose database search index will be triggered for a rebuild
+# By default, this will use the openmrs demo site.
+openmrs.baseUr=https://demo.openmrs.org/openmrs
```

Comment:
Please remove the default value, it doesn't make sense to point it to demo, I suggest you add a property in the receiver that disables the need to rebuild the search index, so the default behavior would be not to rebuild it.

Reply:
Ok, thanks @wluyima. Let me do just that.

Reply:
@wluyima, looking further into the affected route, I see this conditional search index rebuilding handled. The dynamic interpolation of these properties remains necessary since that is done before its logic is processed.